resources/event-sources-controller-manager/values.yaml

```diff
@@ -0,0 +1,11 @@
+global:
+  containerRegistry:
+    path: eu.gcr.io/kyma-project
+  controller_manager:
```

Comment:
is this naming convention consistent?

Reply:
Good question, I used the event-bus as an example.

Reply:
I went digging into Kubernetes' official charts and [everything uses camelCase](https://github.com/helm/charts/blob/master/stable/artifactory-ha/values.yaml), so I'll use the same convention. Thanks for pointing that out!